src/main/java/de/dhbw/mh/slang/codegen/JavaBytecodeGenerator.java

```diff
@@ -159,6 +159,8 @@ public String visitPost( AstBinaryOperation node, String lhs, String rhs ){
                 sb.append("#" + labelCounter + "_ge:%n");
                 sb.append("iconst_0%n");
                 sb.append("#" + labelCounter + "_end:%n");
+
+				labelCounter++;
```

Comment:
This is an addition to the previous task. Normally, you should make sure to strictly separate features. However, I don't want to be too picky and let the one-liner pass as a stowaway for once.